src/fcntl.rs

```diff
@@ -889,3 +889,6 @@ pub fn posix_fallocate(fd: RawFd, offset: libc::off_t, len: libc::off_t) -> Resu
     }
 }
 }
+
+#[cfg(all(feature="fs", any(target_os = "android", target_os = "linux")))]
```

Comment:
```rust
#[cfg_attr(all(
    feature = "fs",
    any(
        target_os = "android",
        all(target_os = "linux", not(target_env = "musl"))
    )
))]
```
Since this is not available on `musl`, this configuration should work